README.CentOS

```diff
@@ -0,0 +1,43 @@
+## Quickstart on CentOS7 ##
+
+
+Note: This assumes an Infrastructure Server install for x64. The additional 
+dependency packages that need to be installed may vary if you are starting with
+a different base system.
+
+
+Install Dependencies:
+
+sudo yum update
+sudo yum install epel-release
+sudo yum install cmake gcc-c++ libevent-devel golang autoconf pkgconfig \
+    json-c-devel gflags-devel glog-devel protobuf-devel leveldb-devel \
+    openssl-devel gperftools-devel protobuf-compiler sqlite-devel ant \
+    java-1.8.0-openjdk-devel protobuf-java python-gflags protobuf-python \
+    python-ecdsa python-mock python-httplib2 git ldns-devel automake \
+    libtool shtool libunwind-devel
+
+
+
+At this point you should be able to follow the GClient build procedure as 
+listed in README.md. This is recommended as it avoids the need to manually build
+dependencies.
+
+When you issue the GClient sync command you may need to set compiler options
+in order to build successfully. If the build fails to work try using:
+
+CXXFLAGS="-O2 -Wno-error=unused-variable" gclient sync -R
```

Comment:
Could pop these command lines into preformatted block:
```
```bash
CXXFLAGS=... blah -R
``
(actually need 3 backticks, but I can't seem to escape it here)
```